spec/lucky/force_ssl_handler_spec.cr

```diff
@@ -53,6 +53,24 @@ describe Lucky::ForceSSLHandler do
       context.response.status_code.should eq 200
       context.response.headers["Location"]?.should be_nil
     end
+
+    it "enables HSTS with 1 year max age and subdomains by default" do
+      context = build_context(path: "/path")
+
+      run_force_ssl_handler context
+
+      context.response.headers["Strict-Transport-Security"].should eq "max-age=31536000; includeSubDomains"
+    end
```

Comment:
I think that HSTS should be enabled if ForceSSL is on, but I think it should be visible to people so it is easy to discover and configure. 

So instead of setting the default here, what if we set it to `nil` but add comments here: https://github.com/luckyframework/lucky_cli/blob/6956ffe067cf0e4033767253754139360d2c6f9e/src/web_app_skeleton/config/server.cr.ecr#L26

This would be awesome too because we could include a link to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security and explain a bit about it

Once this is merged I can go ahead and rework those comments to mention this

Reply:
Since forcessl is enabled by default, the hsts should be as well. If you set it to `nil`, then it will be disabled. We can set the default config in the lucky_cli to nil and forcessl to `Env.production?` Then just adding the comments on how to configure plus updating the website should be good.

Reply:
ForceSSL isn't enabled by default. It is disabled and has comments on how to enable it. I think this is reasonable since a lot of hobby type sites don't have SSL. I think most frameworks take this approach (making it optional).

With that said, I think yore's right that if force ssl is enabled, it should also have the header. I do think we should document in the class documentation at the top of ForceSslHandler so people know what the defaults are when looking at the API docs (which still need to be finished :P)